properties/pom.xml

```diff
@@ -32,8 +32,9 @@
             <artifactId>quarkus-config-yaml</artifactId>
         </dependency>
         <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-consul-config</artifactId>
+            <groupId>io.quarkiverse.config</groupId>
+            <artifactId>quarkus-config-consul</artifactId>
+            <version>1.0.2</version>
```

Comment:
I think that this version value should be on the main pom.xml properties

Reply:
+1 to parametrize this in the root pom.xml

Reply:
Since it is the only usage of this artefact, I assume that this change will bring us more harm, than good, but it's done.

Reply:
best practice is to have versions defined in one place - root pom.xml